### PR TITLE
fix: Change sidebar icon to zap

### PR DESCRIPTION
**Pull request by ChaituVR**
### Summary

Discussion: https://discord.com/channels/955773041898573854/1030773050217615440/1282637612947079169

Changes home icon in sidebar

### How to test

1. Go to any page except home page

| Before | After |
| --- | --- |
| Untitled 5 | Untitled 5 |
| Untitled 7 | Untitled 4 |

### CODE REVIEW

**Comment by bonustrack** [comment]
No I mean the opposite, keep the current icon on top left, but use the same one on My menu for home

**Review by wa0x6e — APPROVED** [review]
utAck, I like this better, it's more consistent with the icon the in the `my-home` root

**Comment by ChaituVR** [comment]
> No I mean the opposite, keep the current icon on top left, but use the same one on My menu for home

But it is actually a stop icon, don't have any relation to home or snapshot
Untitled 9

**Comment by wa0x6e** [comment]
I always see a 5 yen coin https://en.wikipedia.org/wiki/5_yen_coin

**Comment by Sekhmet** [comment]
Maybe we put our Zap there:
image

That stop sign was burned into my memory so much already that I interpret it as our new logo 😆

**Comment by bonustrack** [comment]
Ok let's try with the zap, just slightly (2-4px) bigger than on the Wik screenshot

**Comment by Sekhmet** [comment]
```diff
diff --git a/apps/ui/src/components/App/Sidebar.vue b/apps/ui/src/components/App/Sidebar.vue
index ed7b387b..3ffed35b 100644
--- a/apps/ui/src/components/App/Sidebar.vue
+++ b/apps/ui/src/components/App/Sidebar.vue
@@ -9,8 +9,29 @@ const followedSpacesStore = useFollowedSpacesStore();
   <div
     class="w-[72px] flex flex-col border-r fixed left-0 inset-y-0 text-center h-screen"
   >
-    <router-link :to="{ name: 'my-home' }" class="h-[72px] block">
-      <IH-stop class="inline-block my-4 size-[32px] text-skin-link" />
+    <router-link
+      :to="{ name: 'my-home' }"
+      class="h-[72px] flex items-center justify-center"
+    >
+      <svg
+        width="36"
+        height="36"
+        viewBox="0 0 256 256"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g clip-path="url(#clip0_89_656)">
+          <path
+            d="M196.993 117.038C196.315 115.213 194.579 114 192.638 114H136.577L168.914 50.804C169.93 48.816 169.406 46.3847 167.655 45.0033C166.81 44.3313 165.794 44 164.782 44C163.7 44 162.618 44.3827 161.75 45.134L127.64 74.744L59.6103 133.801C58.1432 135.075 57.6139 137.133 58.2917 138.957C58.9696 140.782 60.7013 142 62.642 142H118.703L86.3662 205.196C85.3495 207.184 85.8741 209.615 87.6244 210.997C88.4694 211.669 89.4861 212 90.4982 212C91.58 212 92.6617 211.617 93.5299 210.866L127.64 181.256L195.67 122.199C197.141 120.925 197.666 118.867 196.993 117.038Z"
+            fill="white"
+          />
+        </g>
+        <defs>
+          <clipPath id="clip0_89_656">
+            <rect width="256" height="256" fill="white" />
+          </clipPath>
+        </defs>
+      </svg>
     </router-link>
     <div
       class="bg-gradient-to-b from-skin-bg top-[72px] h-[8px] w-[71px] absolute z-10"
```

I used this SVG (resized to 36 now).

**Comment by bonustrack** [comment]
I think it's better to store the svg in our repo assets

**Comment by Sekhmet** [comment]
Agreed, just shared SVG to be added in this PR.

**Comment by ChaituVR** [comment]
Changed it to zap 
Untitled 4
Untitled 5

File: apps/ui/src/components/App/Sidebar.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@ const followedSpacesStore = useFollowedSpacesStore();
     class="w-[72px] flex flex-col border-r fixed left-0 inset-y-0 text-center h-screen"
   >
     <router-link :to="{ name: 'my-home' }" class="h-[72px] block">
-      <IH-stop class="inline-block my-4 size-[32px] text-skin-link" />
+      <IC-zap class="inline-block my-4 size-[36px] text-skin-link" />
```
Comment (bonustrack) [inline apps/ui/src/components/App/Sidebar.vue:13]: Like this, a bit bigger and centered with the topnav
```suggestion
      <IC-zap class="inline-block my-[18px] [ size-[40px] text-skin-link" />
```

File: apps/ui/src/components/App/Sidebar.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@ const followedSpacesStore = useFollowedSpacesStore();
     class="w-[72px] flex flex-col border-r fixed left-0 inset-y-0 text-center h-screen"
   >
     <router-link :to="{ name: 'my-home' }" class="h-[72px] block">
-      <IH-stop class="inline-block my-4 size-[32px] text-skin-link" />
+      <IC-zap class="inline-block my-[18px] [ size-[40px] text-skin-link" />
```
Comment (ChaituVR) [inline apps/ui/src/components/App/Sidebar.vue:13]:
```suggestion
      <IC-zap class="inline-block my-[18px] size-[40px] text-skin-link" />
```

**Review by bonustrack — APPROVED** [review]
tACK

**Pull request merged**